internal/naming/names.go

```diff
@@ -186,6 +186,11 @@ const (
 	RestoreConfigCopySuffix = "%s-restorecopy-%d"
 )
 
+const (
+	// AppProtocolTCP is the name of the appProtocol for the Service which use the http protocol
+	AppProtocolHTTP = "http"
```

Comment:
⛏️ The comment says AppProtocolTCP but the name is AppProtocolHTTP.

```suggestion
	// AppProtocolHTTP is the name of the appProtocol for Services which use the HTTP protocol
	AppProtocolHTTP = "http"
```

Reply:
Made this change and attempted to squash the commits, running `make check-envtest` passes locally